docs/basic-features/built-in-css-support.md

```diff
@@ -157,10 +157,19 @@ export default HelloWorld
 
 Please see the [styled-jsx documentation](https://github.com/zeit/styled-jsx) for more examples.
 
-## Sass, Less, and Stylus Support
+## Sass Support
 
-To support importing `.scss`, `.less` or `.styl` files you can use the following plugins:
+To enable importing global `.scss` or `.sass` within `pages/_app.js` and component-level CSS Modules with `.module.scss` or `.module.sass` install `node-sass` as a dependency to your project:
+
+```bash
+npm install node-sass
```

Comment:
This should be just `sass`:
```suggestion
npm install sass
```

Reply:
`sass` is Dart Sass compiled to JS while `node-sass` is a binding to LibSass the C version, which makes it much faster. `sass-loader` also prefers `node-sass` over Dart if both are installed for that reason too.

Reply:
Yup, I think using the pure JS version is better from an ergonomics / first time user standpoint. node gyp is notoriously verbose and troublesome for beginners. I'd expect advance users to eventually dig deep into this and swap out their dependency, or already know to install `node-sass` instead.